### DISCUSSION TITLE
Why the HTTP TASK is not work well ???

### DISCUSSION THREAD

**Discussion by xiaowei582648206** [Netflix/conductor#3468] [Q&A]
I created a workflow that contains a **INLINE_TASK** and **HTTP_TASK**,  the status  HTTP_TASK is always  **SCHEDULED** , just as shown in the picture below：
<img width="1766" alt="image" src="https://user-images.githubusercontent.com/5030860/215303547-d0a4ae1c-32df-4239-a448-9904ccd6367c.png">

the definition of workflow is： 
`{
  "ownerApp": "jmr",
  "createTime": 0,
  "updateTime": 1674962415151,
  "createdBy": "xiaowei12",
  "updatedBy": "xiaowei12",
  "accessPolicy": {},
  "name": "simple_demo_wf_for_xiaowei12",
  "description": "just a simple demo",
  "version": 0,
  "tasks": [
    {
      "name": "inline_task_example",
      "taskReferenceName": "inline_task_example",
      "inputParameters": {
        "value": "${workflow.input.age}",
        "evaluatorType": "javascript",
        "expression": "function e() { if ($.value == 1){return {\"result\": true}} else { return {\"result\": false}}} e();"
      },
      "type": "INLINE",
      "startDelay": 0,
      "optional": false,
      "asyncComplete": false
    },
    {
      "name": "Get Example",
      "taskReferenceName": "get_example",
      "inputParameters": {
        "http_request": {
          "uri": "https://www.baidu.com/",
          "method": "GET"
        }
      },
      "type": "HTTP",
      "startDelay": 0,
      "optional": false,
      "asyncComplete": false
    }
  ],
  "inputParameters": [
    "name",
    "age"
  ],
  "outputParameters": {},
  "schemaVersion": 2,
  "restartable": true,
  "workflowStatusListenerEnabled": true,
  "ownerEmail": "xiaowei79@jd.com",
  "timeoutPolicy": "TIME_OUT_WF",
  "timeoutSeconds": 0,
  "variables": {},
  "inputTemplate": {}
}`

the inputParameters of workflow is:
`{
  "name": "xiaowei79",
   "age": 22
}`

the **JSON** of HTTP TASK is :
`{
  "taskType": "HTTP",
  "status": "SCHEDULED",
  "inputData": {
    "asyncComplete": false,
    "http_request": {
      "method": "GET",
      "uri": "https://www.baidu.com/"
    }
  },
  "referenceTaskName": "get_example",
  "retryCount": 0,
  "seq": 2,
  "pollCount": 0,
  "taskDefName": "Get Example",
  "scheduledTime": 1674962889269,
  "startTime": 0,
  "endTime": 0,
  "updateTime": 0,
  "startDelayInSeconds": 0,
  "retried": false,
  "executed": false,
  "callbackFromWorker": true,
  "responseTimeoutSeconds": 0,
  "workflowInstanceId": "036883e9-3c6a-4cfe-b48e-69ca9e7b5e4d",
  "workflowType": "simple_demo_wf_for_xiaowei12",
  "taskId": "266bdcc3-8ffd-4a8c-91bd-ff57dfcc6ed1",
  "callbackAfterSeconds": 0,
  "outputData": {},
  "workflowTask": {
    "name": "Get Example",
    "taskReferenceName": "get_example",
    "inputParameters": {
      "http_request": {
        "uri": "https://www.baidu.com/",
        "method": "GET"
      },
      "asyncComplete": false
    },
    "type": "HTTP",
    "startDelay": 0,
    "optional": false,
    "asyncComplete": false
  },
  "rateLimitPerFrequency": 0,
  "rateLimitFrequencyInSeconds": 0,
  "workflowPriority": 0,
  "iteration": 0,
  "subworkflowChanged": false,
  "taskDefinition": null,
  "queueWaitTime": 0,
  "loopOverTask": false
}`

the version of conductor-server is : **v3.13.2**.

can you help me find the problem?

**Comment by autodidactic** [comment]
i see the same issue in 3.13.5 version

workflow is running but task is scheduled. 
the api call works. 
its created from new workflow definition in ui

{
  "ownerApp": "",
  "createTime": 1680633560008,
  "status": "RUNNING",
  "endTime": 0,
  "workflowId": "b233cc5a-7a02-4d53-a737-0698feb2a2c4",
  "tasks": [
    {
      "taskType": "HTTP",
      "status": "SCHEDULED",
      "inputData": {
        "asyncComplete": false,
        "http_request": {
          "method": "GET",
          "uri": "https://datausa.io/api/data?drilldowns=Nation&measures=Population"
        }
      },
      "referenceTaskName": "get_population_data",
      "retryCount": 0,
      "seq": 1,
      "pollCount": 0,
      "taskDefName": "get_population_data",
      "scheduledTime": 1680633560557,
      "startTime": 0,
      "endTime": 0,
      "updateTime": 0,
      "startDelayInSeconds": 0,
      "retried": false,
      "executed": false,
      "callbackFromWorker": true,
      "responseTimeoutSeconds": 0,
      "workflowInstanceId": "b233cc5a-7a02-4d53-a737-0698feb2a2c4",
      "workflowType": "Conduc_3.13.5",
      "taskId": "0ec48ff3-0e77-455c-9f73-31f5e2e23f81",
      "callbackAfterSeconds": 0,
      "outputData": {},
      "workflowTask": {
        "name": "get_population_data",
        "taskReferenceName": "get_population_data",
        "inputParameters": {
          "http_request": {
            "uri": "https://datausa.io/api/data?drilldowns=Nation&measures=Population",
            "method": "GET"
          },
          "asyncComplete": false
        },
        "type": "HTTP",
        "startDelay": 0,
        "optional": false,
        "asyncComplete": false
      },
      "rateLimitPerFrequency": 0,
      "rateLimitFrequencyInSeconds": 0,
      "workflowPriority": 0,
      "iteration": 0,
      "subworkflowChanged": false,
      "taskDefinition": null,
      "queueWaitTime": 0,
      "loopOverTask": false
    }
  ],
  "input": {},
  "output": {},
  "taskToDomain": {},
  "failedReferenceTaskNames": [],
  "workflowDefinition": {
    "createTime": 1680633455822,
    "name": "Conduc_3.13.5",
    "description": "conductor workflow for 3.13.5",
    "version": 1,
    "tasks": [
      {
        "name": "get_population_data",
        "taskReferenceName": "get_population_data",
        "inputParameters": {
          "http_request": {
            "uri": "https://datausa.io/api/data?drilldowns=Nation&measures=Population",
            "method": "GET"
          },
          "asyncComplete": false
        },
        "type": "HTTP",
        "startDelay": 0,
        "optional": false,
        "asyncComplete": false
      }
    ],
    "inputParameters": [],
    "outputParameters": {
      "data": "${get_population_data.output.response.body.data}",
      "source": "${get_population_data.output.response.body.source}"
    },
    "schemaVersion": 2,
    "restartable": true,
    "workflowStatusListenerEnabled": false,
    "ownerEmail": "example@email.com",
    "timeoutPolicy": "ALERT_ONLY",
    "timeoutSeconds": 0,
    "variables": {},
    "inputTemplate": {}
  },
  "priority": 0,
  "variables": {},
  "lastRetriedTime": 0,
  "failedTaskNames": [],
  "workflowVersion": 1,
  "workflowName": "Conduc_3.13.5",
  "startTime": 1680633560008
}

**Comment by v1r3n** [comment]
Do you have system tasks workers enabled?

**Reply by autodidactic** [reply]
i am sure i missed something.. what kind of system task are we talking about  .HTTP task? how do i enable them? i would imagine from the ui there is an indication that says i need to do this in order to execute "HTTP" type task.
any documentation would be helpful

thank you

**Reply by viinxent** [reply]
Have you fixed this? if so what is your solution? thanks!
